vignettes/b1-setting-up-and-using-rix-on-linux-and-windows.Rmd

```diff
@@ -86,14 +86,18 @@ and then relaunch WSL (Ubuntu) from the start menu.
 Afterwards, you can install Nix like business as usual. You can proceed with the
 Determinate Systems installer.
 
-### The Determinate Systems installer
+### Use the Determinate Systems installeron Linux and Windows
```

Comment:
keep the title as it was before, as the determinate systems installer also works on macOS